cli/tests/pcluster/schemas/test_schema_validators.py

```diff
@@ -590,7 +591,9 @@ def test_subnet_id_validator_aws_batch(subnet_ids, expected_message):
         (["NONE"], "does not match expected pattern"),
         (["subnet-12345678"], None),
         (["subnet-12345678901234567"], None),
-        (["subnet-1234", "subnet-5678"], None),
+        (["subnet-12345678901234568"], "does not match expected pattern"),
```

Comment:
Actually, this is wrong: these are still 17 chars in the subnet id, so that should be a valid ID.

Please use `subnet-123456789012345678` for this test.